src/Abc.Zebus.MessageDsl.Tool.Tests/ToolTests.cs

```diff
@@ -0,0 +1,94 @@
+using NUnit.Framework;
+
+namespace Abc.Zebus.MessageDsl.Tool.Tests;
+
+public class ToolTests
+{
+    private TextWriter _originalOut = null!;
+    private TextReader _originalIn = null!;
+
+    [SetUp]
+    public void Setup()
+    {
+        _originalOut = Console.Out;
+        _originalIn = Console.In;
+    }
+
+    [TearDown]
+    public void TearDown()
+    {
+        Console.SetOut(_originalOut);
+        Console.SetIn(_originalIn);
+    }
+
+    [Test]
+    public void should_generate_csharp_with_namespace()
+    {
+        string inputContent = "TestMessage";
```

Comment:
That's not exactly a valid input... `TestMessage()` would be valid.

It looks like the tool generates output for invalid input. That's what we want for a Roslyn source generator, but the command-line tool should error out if the input is invalid.

See `ParsedContracts.IsValid` and `GenerateZebusMessagesTask` which prints all the errors.